tests/test_crypto/test_ethereum.py

```diff
@@ -18,240 +18,10 @@
 # ------------------------------------------------------------------------------
 """This module contains the tests of the ethereum module."""
 
-import hashlib
-import logging
-import time
-from pathlib import Path
-from unittest.mock import MagicMock, patch
-
 import pytest
+from ethereum_crypto import EthereumApi
```

Comment:
Why are parts of this file staying?

Reply:
beause we are testing the ERC1155 contract, which is not a test for the plugin `ethereum_crypto`.